weblate/static/style-bootstrap.css

```
@@ -551,6 +551,11 @@ form + .login-label {
   height: 20px;
 }
 
+#information .table {
```

Shouldn't this apply to all tables embed in the panel?

```suggestion
.panel > .table {
```

---

No. `display: block;` has a weird effect on other tables that take all width of the screen. It makes them smaller. And the width property doesn't take effect.